entrypoint.sh

```diff
@@ -1,4 +1,4 @@
 #!/bin/bash
 
 echo "Loading fixtures."
-python /code/manage.py loaddata questions.json || exit 1
\ No newline at end of file
+python3 /code/manage.py loaddata questions.json || exit 1
```

Comment:
What is the reason for changing this to python3?

Reply:
Not too sure, I don't believe I did this